kolibri/plugins/setup_wizard/assets/src/state/store.js

```diff
@@ -1,17 +1,62 @@
 import Vuex from 'kolibri.lib.vuex';
+import { findKey } from 'lodash';
```

Comment:
Does importing in this destructured way properly jettison the rest of lodash? One would hope that tree-shaking would sort this out for is, but would be nice to verify.

Reply:
in other places we use this pattern:

```javascript
import findKey from 'lodash/findKey';
```

Reply:
I'd verify if I knew how - any tips on how to go about it?